Automattic/wp-calypso
# Jetpack Connect: add Jetpack branding to plans grid

#### Changes proposed in this Pull Request

* Adds Jetpack logo to plans grid.
* Adds Jetpack brand colors to buttons.

#### Testing instructions

* Start on `/jetpack/connect/store/`
* Make sure you see the Jetpack logo and the Jetpack brand colors on the plans grid.

#### Before

![image](https://user-images.githubusercontent.com/390760/49144373-1fd8a080-f2f5-11e8-99dd-90b7784fd812.png)

#### After

![image](https://user-images.githubusercontent.com/390760/49144385-2c5cf900-f2f5-11e8-9d25-1a38de0d59ee.png)

Fixes #28419
